spec.html

```diff
@@ -28657,8 +28657,6 @@ <h1>Runtime Semantics: GetSubstitution ( _matched_, _str_, _position_, _captures
             1. Assert: Type(_replacement_) is String.
             1. Let _tailPos_ be _position_ + _matchLength_.
             1. Let _m_ be the number of elements in _captures_.
-            1. If _namedCaptures_ is not *undefined*, then
-              1. Set _namedCaptures_ to ? ToObject(_namedCaptures_).
```

Comment:
This would be an observable/normative change, since the order the exception is thrown matters.